### ISSUE TITLE
make test-cov fails, possibly due to reorganized NIST OSCAL website

### ISSUE THREAD

**Issue by degenaro** [IBM/compliance-trestle#1434]
## Describe the bug
```

[gw8] [ 22%] FAILED tests/trestle/core/profile_resolver_test.py::test_remote_profile_relative_cat 

================================================================================= FAILURES =================================================================================
_____________________________________________________________________ test_remote_profile_relative_cat _____________________________________________________________________
[gw8] linux -- Python 3.9.9 /home/degenaro/venv.special/bin/python

self = <trestle.core.remote.cache.HTTPSFetcher object at 0x7f899cf74fd0>, force_update = False

    def _update_cache(self, force_update: bool = False) -> bool:
        """Update the cache by fetching the target remote object, if stale or forced.
    
        Args:
            force_update: force the fetch regardless of staleness.
    
        Returns:
            True if update occurred
        """
        if self._is_stale() or force_update:
            try:
>               self._do_fetch()

/home/degenaro/git/trestle.feat-special-validation-component-handling/trestle/core/remote/cache.py:100: 
```

## To Reproduce

1. download trestle develop branch
1. create venv
1. make install
1. make develop
1. make test-cov

## Expected behavior

Test failure should not occur.

Temporary bypass: remove `test_remote_profile_relative_cat` from `tests/trestle/core/profile_resolver_test.py`.

## Environment

- OS: Red Hat 
- Python version: 3.9
- Installed packages: trestle

**Comment by AleJo2995** [comment]
Thanks for the issue creation @degenaro . I will be reviewing it and taking care of it.

**Comment by degenaro** [comment]
I have a fix for this. I'll go ahead and make a branch and deliver. This is kind of a hot fix, since builds for PRs fail.

**Comment by degenaro** [comment]
Fixed by #1435 .

**Comment by AleJo2995** [comment]
Thanks @degenaro
